R/findNLDI.R

```diff
@@ -226,52 +240,46 @@ valid_ask = function(all, type){
 #' ## GENERAL ORIGIN: WaDE
 #'  findNLDI(origin = list("wade" = 'CA_45206'))
 #'
-#' # Navigation
+#' # Navigation (flowlines will be returned by default)
 #' # UPPER MAINSTEM of USGS-11120000
-#'  str(findNLDI(nwis = '11120000', nav = "UM"), max.level = 1)
+#'  findNLDI(nwis = '11120000', nav = "UM")
 #'
 #' # MULTI-REQUEST
 #' # UPPER MAINSTEM and TRIBUTARY of USGS-11120000
-#'  str(findNLDI(nwis = '11120000', nav = c("UT", "UM")), max.level = 1)
+#'  findNLDI(nwis = '11120000', nav = c("UT", "UM"))
 #'
-#' # Discover Features
+#' # Discover Features(flowlines will not be returned by default)
 #'
 #' ## Find feature(s) on the upper tributary of USGS-11120000
-#'  str(findNLDI(nwis = '11120000', nav = "UT", find = c("nwis", "wqp")), max.level = 1)
+#'  findNLDI(nwis = '11120000', nav = "UT", find = c("nwis", "wqp")))
 #'
-#' ## Find upstream basin boundary of USGS-11120000
-#'  str(findNLDI(nwis = '11120000',  find = "basin"), max.level = 1)
+#' ## Find upstream basin boundary and  of USGS-11120000
+#'  findNLDI(nwis = '11120000',  find = "basin")
 #'
 #' # Control Distance
-#'
-#' ## Limit search to 100 km
-#'  str(findNLDI(comid = 101, nav = "DM", find = c("nwis", "wqp"), distance_km = 100), max.level = 1)
-#'
-#' ## Convert returned list of data.frames to list of spatial features
-#' \donttest{
-#'  nldi = findNLDI(nwis = '11120000', nav = "UT", find = c("nwis", "wqp"))
-#'  str(nldi, max.level = 1)
-#'  }
+#' ## Limit search to 50 km
+#'  findNLDI(comid = 101, nav = "DM", find = c("nwis", "wqp", "flowlines"), distance_km = 50)
 
 
+location = AOI::geocode("UCSB", pt = TRUE)
```

Comment:
Gotta drop this call to AOI. Tests are seeing it and puking.

Reply:
Wow. Haha, nice catch!